packages/terra-base/package.json

```diff
@@ -28,8 +28,7 @@
     "react-dom": "^15.4.2"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
-    "react": "^15.4.2"
```

Comment:
I have some reservations about removing react as a dependency. Let's talk this some more tomorrow